addons/ui/functions/fnc_setAdvancedElement.sqf

```diff
@@ -66,6 +66,7 @@ if (!_force) then {
     };
 };
 
+private _displays = (uiNamespace getVariable "IGUI_displays") + [findDisplay IDD_MISSION] select {_idd == ctrlIDD _x};
```

Comment:
Does `+` have precedence over `select`? Otherwise this doesn't do the same thing as it used to.

Reply:
yes

Reply:
Would prefer some parentheses to make that clear then.